### ISSUE TITLE
NPC conversations being heard by other players

### ISSUE THREAD

**Issue by iSkitzu** [CitizensDev/Citizens2#1282]
In this issue, please include:
- whether this is a bug description or a feature request
- Link to server log & Citizens config if relevant (using e.g. pastie.org) 
- Minecraft & Citizens version if relevant

Bug Report(?)

[19:42:18] [INFO] =====[ Citizens v2.0.22-SNAPSHOT (build 1487) ]=====
[19:42:22] [INFO] This server is running CraftBukkit version git-Spigot-65e8124-79e55b6 (MC: 1.12.1) (Implementing API version 1.12.1-R0.1-SNAPSHOT)
[19:42:22] [INFO] Checking version, please wait...
[19:42:22] [INFO] You are running the latest version

I wanted to know if there is a way to avoid surrounding players from hearing npcs talk to a player who clicked on them.

In other words, when I click on an NPC, other players within about 5 blocks can also hear that npc. Considering there are npcs in crowded areas, I don't want their chats to get clocked with people talking to npcs around them.

**Comment by mcmonkey4eva** [comment]
If using Denizen to back your Citizens NPC chatter, use `narrate` instead of `chat`.
If not using Denizen... use Denizen :P
